test/typed-function-references.wast.fromBinary

```diff
@@ -93,7 +93,7 @@
    )
   )
  )
- (func $nested-type-only-there (result (ref null (func (result (ref null (func (param i32 i32 i32 i32 i32)))))))
+ (func $nested-type-only-there (result (ref null $none_=>_ref_null[i32_i32_i32_i32_i32_->_]))
```

Comment:
What would you think of using `?` instead of `_null` to make the names shorter? (I forget if '?' is even an allowed character in names...)

Reply:
Oh, nice, I prefer `?` -  good idea.

I think it's valid. Anything but `()"'` should be fine IIRC.